test/e2e/specs/site-editor/navigation.spec.js

```diff
@@ -106,6 +106,25 @@ test.describe( 'Site editor navigation', () => {
 		// We should have our editor canvas button back
 		await expect( editorCanvasButton ).toBeVisible();
 	} );
+
+	test( 'Should show 404 page when navigating to non-existent template', async ( {
+		admin,
+		page,
+	} ) => {
+		await admin.visitSiteEditor();
+
+		// Navigate to a non-existent template.
```

Comment:
```suggestion
		// Navigate to a non-existent template.
```

Nit: `visitSiteEditor()` is equivalent to `visitAdminPage( 'site-editor.php' )`, which I think can be removed since there is already a subsequent `visitAdminPage`.

Reply:
I think that makes sense. What was the reason for calling `visitSiteEditor`?